www/install/php/Update-20.10.3.php

```diff
@@ -59,7 +60,36 @@
         }
     }
 } catch (\Throwable $ex) {
-    require_once __DIR__ . '/../../class/centreonLog.class.php';
+    (new CentreonLog())->insertLog(
+        4,
+        $versionOfTheUpgrade . $errorMessage .
+        " - Code : " . $ex->getCode() .
+        " - Error : " . $ex->getMessage() .
+        " - Trace : " . $ex->getTraceAsString()
+    );
+    throw new \Exception($versionOfTheUpgrade . $errorMessage, $ex->getCode(), $ex);
+}
+
+// Contact language with transaction
+try {
```

Comment:
More simple with this SQL request:
`UPDATE contact SET contact_lang = CONCAT(contact_lang, '.UTF-8') WHERE contact_lang NOT LIKE "%.UTF-8";`